TF_Service_dotNet/TouchFree/Connections/ClientConnectionManager.cs

```diff
@@ -41,12 +43,24 @@ public ClientConnectionManager(IHandManager _handManager, IConfigManager _config
 
         private void ConnectionStatusChange(TrackingServiceState state)
         {
-            var currentConfig = new ServiceStatus(
-                string.Empty, // No request id as this event is not a response to a request
-                state,
-                configManager.ErrorLoadingConfigFiles ? ConfigurationState.ERRORED : ConfigurationState.LOADED);
-
-            SendResponse(currentConfig, ActionCode.SERVICE_STATUS);
+            void handleDeviceInfoResponse() 
+            {
+                var currentConfig = new ServiceStatus(
+                    string.Empty, // No request id as this event is not a response to a request
+                    state,
+                    configManager.ErrorLoadingConfigFiles ? ConfigurationState.ERRORED : ConfigurationState.LOADED,
+                    VersionManager.ApiVersion.ToString(),
+                    trackingApi.trackingServiceVersion != null ? trackingApi.trackingServiceVersion : "Tracking not connected",
+                    trackingApi.connectedDeviceSerial != null ? trackingApi.connectedDeviceSerial : "Device not connected",
+                    trackingApi.connectedDeviceFirmware != null ? trackingApi.connectedDeviceFirmware : "Device not connected");
```

Comment:
I feel like because this is duplicated it might be worth pushing into the constructor of ServiceStatus